src/components/Menu.tsx

```diff
@@ -2,11 +2,15 @@ import { FaFish, FaHouse, FaTree } from "react-icons/fa6";
 import { MdHexagon } from "react-icons/md";
 import { GiCardPickup } from "react-icons/gi";
 import { useGameState } from "@/context/game-state/hook";
-import { filter } from "lodash";
 import clsx from "clsx";
+import { useSelector } from "@xstate/react";
+import { MachineSelectors } from "@/state/machines/selectors";
 
 const Menu = () => {
-  const { state, showPolicies, setShowPolicies, gameConfig } = useGameState();
+  const { state, showPolicies, setShowPolicies, gameConfig, actorRef } = useGameState();
+  const numberOfAnimalsBought = useSelector(actorRef, MachineSelectors.numberOfAnimalsBought);
```

Comment:
Are there any reasons we export `MachineSelectors` objects instead of individual selector functions?

Reply:
Mmm no I cant remember why it was done like this, but probably to fit in with the TurnMachineGuards structure

Reply:
I changed them now to individually exported selector functions